build-recipe-kiwi

```diff
@@ -507,6 +507,34 @@ is_legacy_kiwi() {
     return 0
 }
 
+get_kiwi_version() {
+    # lookup version number of installed kiwi version and
+    # turn it into a reliable number
+    local version
+    local major
+    local minor
+    local patch
+    if is_legacy_kiwi; then
```

Comment:
How does it know it's a legacy kiwi? I don't see anything in the function that tells it that it is...

If anything, the simpler thing to do would be the reverse the logic and check for `kiwi-ng` if it exists, and if it does, you know it's not legacy kiwi, then check the version. If the version is high enough, it's good for zsync support check.